Expensify/App
# Use sublocality when locality is missing - AddressSearch

### Details
There are addresses that have the component "locality" missing. We use the "locality" as the city. This PR will make us use the "sublocality" as the city when the "locality" is missing.

### Fixed Issues

$ https://github.com/Expensify/App/issues/5867

### Tests / QA Steps

1. Create account in NewDot, verify it and log in
2. Create Workspace
3. Start add VBA flow and get to the "Company information" step (/bank-account/company)
4. Input in the "Company address" field: `64 Noll Street, Brooklyn, NY, USA` and select the first (and only) result
5. Fill the rest of the fields
6. You should be able to continue to the next step without errors.

### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
